package.json

```diff
@@ -8,10 +8,14 @@
   "license": "MIT",
   "scripts": {
     "install:nx": "npm install --global nx@latest",
-    "setup": "npm install && npx prisma generate --schema apps/api/prisma/schema.prisma",
+    "db:setup": "node migrate_and_seed_database.js",
+    "setup": "npm install && npx prisma generate --schema apps/api/prisma/schema.prisma && npm run db:setup",
+    "test": "NODE_ENV=test nx run-many --target=test --all",
+    "test:api": "NODE_ENV=test nx test api",
+    "test:client": "NODE_ENV=test nx test client",
     "start:api": "nx run api:serve:development",
     "start:client": "nx run client:serve:development",
-    "start": "npm run start:api & npm run start:client"
+    "start": "concurrently \"npm run start:api\" \"npm run start:client\""
```

Comment:
By using concurrently, we can start both servers and still correctly handle signals to ensure clean application shutdown.